test/modelValidatorTests.ts

```diff
@@ -11,7 +11,7 @@ const testPath = __dirname;
 const specPath =
   `${testPath}/modelValidation/swaggers/specification/scenarios/resource-manager/` +
   `Microsoft.Test/2016-01-01/test.json`;
-
+jest.setTimeout(999999);
```

Comment:
Will this need when run in pipeline or just need when run locally?

Reply:
This is needed in pipeline。
Sometimes the test "should pass when specPath is a url starting with https://github/" will report a timeout error in pipeline.
Should I only change timeout of the test instead of set all timeout to "999999"?